### PR TITLE
[BOOTDATA] Show Explorer status bar by default

### PR DESCRIPTION
**Pull request by cbialorucki**
## Purpose

Enable the explorer status bar by default, which the vast majority of our community voted for.

JIRA issues: [CORE-19010](https://jira.reactos.org/browse/CORE-19010) [CORE-19028](https://jira.reactos.org/browse/CORE-19028)

## Proposed changes

- Set ```HKCU\Software\Microsoft\Internet Explorer\Main\StatusBarOther``` to 1 in ```hivedef.inf```. This will show the status bar by default.

### CODE REVIEW

**Review by katahiromz — APPROVED** [review]

**Review by Doug-Lyons — CHANGES_REQUESTED** [review]
You should probably insert a blank line below your new one to separate the list that is described by '; Current user shell folder settings' from '; ComDlg32'. Thanks for your efforts.

**Review by JoachimHenze — APPROVED** [review]

**Comment by JoachimHenze** [comment]
yes, a blank line before and after the added line would make sense. Good point Doug.

**Review by KRosUser2 — APPROVED** [review]
lgtm

File: boot/bootdata/hivedef.inf [hunk 1]
```diff
@@ -1899,7 +1899,7 @@ HKCU,"SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\Advanced","StartMenuLog
 ; "Hidden" to be changed to 2 if we later want to have "Hide hidden files by default"
 HKCU,"SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\Advanced","Hidden",0x00010003,1
 HKCU,"SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\Advanced","ShowSuperHidden",0x00010003,0
-
+HKCU,"SOFTWARE\Microsoft\Internet Explorer\Main","StatusBarOther",0x00010003,1
 ; ComDlg32
```
Comment (HBelusca) [inline boot/bootdata/hivedef.inf:1903]: Adding back the separating newline that originally existed.
```suggestion

; ComDlg32
```

**Review by binarymaster — APPROVED** [review]
Just a note, once this will be applied, it will be impossible to reproduce another uncovered bug [CORE-19027](https://jira.reactos.org/browse/CORE-19027) - so ideally it needs to be fixed first.

**Comment by cbialorucki** [comment]
> Just a note, once this gets merged, it will be impossible to reproduce another uncovered bug [CORE-19027](https://jira.reactos.org/browse/CORE-19027) - so ideally it needs to be fixed first.

Thanks to status bar persistence, I can still reproduce this bug by disabling the status bar, closing the explorer window, reopening it, then opening the folder pane. This also affects the search pane.

**Comment by binarymaster** [comment]
> I can still reproduce this bug by disabling the status bar, closing the explorer window, reopening it, then opening the folder pane. This also affects the search pane.

Indeed I can confirm this as well, so nothing blocks this improvement. Thank you! 🙂

**Pull request merged**